ha/util/conf_store.py

```diff
@@ -221,3 +221,19 @@ def get_disk_list_for_cvg(index, node_id, cvg_id):
         except Exception as e:
             Log.error(f"Unable to fetch Disk list from GConf. Error {e}")
             raise Exception(f"Unable to fetch Disk list. Error {e}")
+
+    @staticmethod
+    def get_control_node_id(index):
```

Comment:
Please correct method name to align with server and data pod get method names.

Reply:
Done.
Renamed the func to get_control_pod() on similar lines with get_data_pods/get_server_pods